src/export/oneApp/index.test.ts

```diff
@@ -0,0 +1,82 @@
+import { X2jOptionsOptional, XMLParser, XMLValidator } from "fast-xml-parser";
+import get from "lodash.get";
+import { OneAppPayload } from "./model";
+import { generateOneAppXML } from "./index";
+import { graphQLClient } from "../../graphql";
+import {
+  FileAttachment,
+} from "./types";
+import { mockSession } from "./mocks/session";
+
+let mockHasRequiredDataForTemplate = jest.fn();
+jest.mock("@opensystemslab/planx-document-templates", () => {
+  return {
+    hasRequiredDataForTemplate: jest.fn(() => mockHasRequiredDataForTemplate()),
+  };
+});
+
+jest.mock("../../session", () => ({
+  getSessionById: jest.fn(() => mockSession)
+}));
```

Comment:
I was just wondering about options for query mocking over here in planx-core actually - this looks good & is helpful to see :+1: